src/harness/fourslashInterfaceImpl.ts

```diff
@@ -239,6 +239,10 @@ export class VerifyNegatable {
     public uncommentSelection(newFileContent: string) {
         this.state.uncommentSelection(newFileContent);
     }
+
+    public mapCode(newFileContent: Record<string, string>, mappings: ts.MapCodeDocumentMapping[], updates?: ts.FileTextChanges[]): void {
+        this.state.mapCode(newFileContent, mappings, updates);

```

Comment:
I don't understand how you're picturing the `=====`-separated sections working. Is there another test/set of tests that do this kind of thing? I don't see any tests that do anything like this.

Reply:
In `fourslashImpl.ts`, most of the test functions with `baseline` in them (and there are some tests that generate baselines that aren't explicitly labeled that way) will have some sort of formatting trick in them. `baselineGoToDefinition` or `baselineRename` could be good to look at for an idea, and I think I did something similar for the linked editing as well.

Reply:
I haven't looked at the tests so far yet, but doing a baseline does make it easier in the long run, especially if you're adding a lot of tests, or there's a lot of output that you want to check, so that you don't have to worry about manually producing the correct test output for every one